bot/plotting/random_plot_generator.py

```diff
@@ -213,6 +213,7 @@ def random_plot_generator(
                     degradation_parameter=degradation_parameter
                 )
 
+                labels = sorted(labels, key=lambda x: float(x.split(':')[1]))
```

Comment:
Does this just sort the labels and not the solutions? I.e. are the labels incorrect now?

Reply:
Yes, the solutions and labels are jumbled up:)